testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -60,3 +60,30 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I click the Add Product button
     And I wait until I see "SUSE Linux Enterprise Server 15 SP2 x86_64" product has been added
     Then the SLE15 products should be added
+
+@ssh_minion
+@scc_credentials
+  Scenario: Add a 15 SP1 product
```

Comment:
Cosmetical: add why you are doing it here

You will also need to sync SLES 15 SP2. I know this is already done elsewhere, but we should not depend on that other code. Maybe add a scenario, or use a tag `@sp_migration`, or add a warning in the code. But find something to protect ourselves from that other code changing.